### PR TITLE
@dblandin => [SearchResults] Add institution to More tab

### PR DESCRIPTION
**Pull request by mzikherman**
Pretty easy! This just adds `INSTITUTION` to the entities for the 'More' tab, as well as tweaks to the tab counts.

### CODE REVIEW

File: src/Apps/Order/Utils/commitMutation.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import {
   GraphQLTaggedNode,
 } from "react-relay"
 import { Environment, OperationBase } from "relay-runtime"
-jest.unmock("react-relay")
```
Comment (mzikherman) [inline src/Apps/Order/Utils/commitMutation.tsx]: @ds300 This was causing some errors so went ahead and removed.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 5764b133bb9b683cc61168b08edcea4cf213676b

https://deploy-preview-2187--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Merging this, it's pretty simple!

**Comment by artsyit** [comment]
:rocket: PR was released in v15.5.13 :rocket:

**Pull request merged**